app/src/main/java/com/rafaelmardom/movies_and_shows_api/domain/GetMoviesFeedUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.rafaelmardom.movies_and_shows_api.domain
+
+class GetMoviesFeedUseCase (private val movieRepository: MovieRepository) {
+
+    fun execute () : List<MovieFeed> {
+        val movies = movieRepository.getMovies()
+
+        return movies.map {
+            MovieFeed(
+                it.id,
+                it.title,
+                it.genre,
+                it.rating,
+                it.poster
+            )
+        }
+    }
+
+}
+
+data class MovieFeed (
```

Comment:
MovieFeed is an application model, please, you should move it inside of use case

Reply:
Done!